.ci/check-standard-images.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# [description]
+#     Checks the saturn* image directories to be sure they
+#     use standard practices. Saturn has some application logic
+#     that relies on the images in this repo as a source of truth
+#     for metadata about images. This script runs on every build to
+#     ensure that the directories are in a state that Saturn will
+#     understand.
+
+set -eou pipefail
+
+echo "----- checking that image metadata follows Saturn practices ----"
+
+images_to_check="
+    saturn
+    saturn-gpu
+    saturnbase
+    saturnbase-gpu
+    "
```

Comment:
I just thought of one thing. If it were easy to put these in a separate file I think that'd be slightly preferable.

Reply:
why would that be preferable?

Reply:
This image list will change in the near future and it feels cleaner to separate inputs from scripts.

Reply:
I disagree but not strongly, so I'll do what you've suggested and then merge this.

In my opinion, it's not worth adding more indirection until something else needs this information.

Reply:
added in https://github.com/saturncloud/images/pull/163/commits/64e83e2ed82539b2b903ebe3bea7a1a1278f9061